distribution-nixpkgs.cabal

```diff
@@ -1,16 +1,19 @@
-name:          distribution-nixpkgs
-version:       1.5.0
-synopsis:      Types and functions to manipulate the Nixpkgs distribution
-description:   Types and functions to represent, query, and manipulate the Nixpkgs distribution.
-license:       BSD3
-license-file:  LICENSE
-maintainer:    Peter Simons <simons@cryp.to>
-tested-with:   GHC == 8.0.2 || == 8.2.2 || == 8.4.4 || == 8.6.5 || == 8.8.4 || == 8.10.4
-category:      Distribution, Nix
-homepage:      https://github.com/peti/distribution-nixpkgs
-bug-reports:   https://github.com/peti/distribution-nixpkgs/issues
-build-type:    Simple
-cabal-version: >= 1.10
+name:                distribution-nixpkgs
```

Comment:
Please don't commit gratuitous white-space changes. Now we have a merge conflict. :-(